src/shell/environment.js

```diff
@@ -1,4 +1,4 @@
-var defaults = require('../constants.js').ENVIRONMENT;
+const defaults = require('../constants.js').ENVIRONMENT;
```

Comment:
Hey, I think you should also include `'use strict';` to the head of the file.

Reply:
good catch! Thanks. :) @vrozin